public/app/app-init.js

```
@@ -1,644 +1,414 @@
-async function ensureAudioContext() {
-  if (audioCtx && audioCtx.state !== 'closed') {
-    if (audioCtx.state === 'suspended') await audioCtx.resume();
+// ─── 4. AUDIO CONTEXT + WORKLET ──────────────────────────────────────────────
+async function initAudio() {
+  if (App.ctx) {
+    if (App.ctx.state === 'suspended') await App.ctx.resume();
     return;
   }
 
-  audioCtx = new AudioContext({ sampleRate: 48000, latencyHint: 'interactive' });
+  App.ctx = new AudioContext({ latencyHint: 'interactive', sampleRate: 48000 });
```

ESLint

[error] 'AudioContext' is not defined.

(no-undef)